src/status_im2/contexts/quo_preview/numbered_keyboard/keyboard_key.cljs

```diff
@@ -0,0 +1,51 @@
+(ns status-im2.contexts.quo-preview.numbered-keyboard.keyboard-key
+  (:require [quo2.core :as quo]
+            [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [reagent.core :as reagent]
+            [status-im2.contexts.quo-preview.preview :as preview]))
+
+
+(def descriptor
+  [{:label "Blur:"
+    :key   :blur?
+    :type  :boolean}
+   {:label "Disable:"
+    :key   :disabled?
+    :type  :boolean}])
+
+(defn cool-preview
+  []
+  (let [state (reagent/atom {:disabled? false
+                             :on-press  #(js/alert "pressed")
+                             :blur?     false})]
+    (fn []
+      [rn/view {:style {:padding-bottom 150}}
+       [rn/view {:style {:flex 1}}
+        [preview/customizer state descriptor]]
+       [preview/blur-view
+        {:style                 {:flex              1
+                                 :margin-horizontal 20}
+         :show-blur-background? (:blur? @state)
+         :blur-view-props       (when (:blur? @state)
+                                  {:overlay-color colors/neutral-80-opa-80})}
+        [rn/view
+         {:style {:flex-direction  :row
+                  :width           220
+                  :padding         20
+                  :justify-content :space-between
+                  :flex            1}}
+         [quo/keyboard-key (assoc @state :type :digit) 1]
+         [quo/keyboard-key (assoc @state :type :key) :i/delete]
+         [quo/keyboard-key (assoc @state :type :digit) "‘/"]]]])))
```

Comment:
this should be `:type :derivation-path` no? 
does this type not exist in the component? 

Also can we make this only one item displayed at a time in the preview screen and add `type` to the `descriptor`?

Reply:
<img width="233" alt="Screenshot 2023-07-27 at 11 24 28" src="https://github.com/status-im/status-mobile/assets/22799766/4e62ec31-824c-4669-9134-0fe9a63e84ae">

Reply:
I will check it with the design team

Reply:
Done